src/Microsoft.Azure.WebJobs.Extensions.PostgreSql/PostgreSqlAsyncCollector.cs

```diff
@@ -4,24 +4,28 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Threading.Channels;
```

Comment:
Sample complete. Added:

- batching commands
- upserts instead of just insert
- primary key / keys support
- add/flush async functionality
- semaphore for concurrency handling
- basic logging